pyroscope_backends/pyroscope_pyspy/.gitignore

```diff
@@ -0,0 +1,5 @@
+/target/
+Cargo.lock
```

Comment:
`Cargo.lock` should become part of the git repository.

Reply:
@drahnr Can you explain the reason for that? My understanding is that Cargo.lock should be included only for the binary. (https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries)